src/hooks/use-webvtt.ts

```diff
@@ -0,0 +1,115 @@
+import { v4 as uuidv4 } from "uuid";
```

Comment:
This is screaming at us when doing `tsc`, not sure how to best handle.

Reply:
Gotcha.  I'll import the `@types/uuid` package as a dev dependency.